toolkit/imageconfigs/packagelists/core-packages-image-aarch64.json

```diff
@@ -1,6 +1,7 @@
 {
     "packages": [
         "shim-unsigned",
+        "grub2-configuration",
```

Comment:
Small nit, do we even want to list this here? Once its here we are sort of stuck with it and in theory the package manager should deal with this. It feels like the underlying mechanisms shouldn't be of interest to the user, just the top level packages.

Reply:
Right, because of the %grub2_configuration_requires macro this should be unnecessary.